lib/ovirt/service.rb

```diff
@@ -199,7 +199,7 @@ def resource_verb(path, verb, *args)
       logger.debug "#{log_header}: With args: <#{args.inspect}>"
       resource.send(verb, *args) do |response, request, result, &block|
         case response.code
-        when 200
+        when 200..206
```

Comment:
@bdunne Is there any special handling required for the return codes in the 200 range or is rest-client dealing with that?